_layouts/redirect.html

```diff
@@ -0,0 +1,24 @@
+---
+---
+<!DOCTYPE html>
```

Comment:
should we add a source comment that this overrides jekyll-redirect_from?
```suggestion
---
---
{% comment %}
  This overrides the template that jekyll_redirect-from uses so that we can make descriptions searchable in the site index
{% endcomment %}
<!DOCTYPE html>
```

Reply:
Yep, we should 👍 Added in 7355547